controllers/postgres_controller.go

```diff
@@ -12,6 +12,7 @@ import (
 	"net/url"
 	"time"
 
+	"github.com/ViaQ/logerr/log"
```

Comment:
This import looks weird

Reply:
Yup, sorry. I hate the auto imports on save in VS Code :-)